packages/fluent-theme/src/fluent/styles/Dialog.styles.ts

```diff
@@ -1,6 +1,8 @@
-import { FontSizes } from '../FluentType';
-import { FontWeights } from 'office-ui-fabric-react/lib/Styling';
+import { IDialogContentStyleProps } from 'office-ui-fabric-react/lib/Dialog';
+import { DefaultPalette, FontWeights } from 'office-ui-fabric-react/lib/Styling';
```

Comment:
@mikewheaton I think the build failed because you have an unused import `DefaultPalette` (auto-imports again 🙄 )

Reply:
Thanks for noticing that! VSCode usually picks up unused imports but it missed this one for some reason.